src/jet/elements/environment.rs

```diff
@@ -19,7 +19,7 @@ use bitcoin_hashes::{sha256, Hash, HashEngine};
 use byteorder::{BigEndian, LittleEndian, WriteBytesExt};
 use elements::confidential::{Asset, Nonce, Value};
 use elements::taproot::ControlBlock;
-use elements::{confidential, AssetIssuance, BlockHash};
+use elements::{confidential, AssetIssuance, BlockHash, Transaction};
```

Comment:
In 158b54dfe7526b3f3c51cbe9d988e46c05c4aa95:

I really find it hard to read `Transaction` without either a `bitcoin::` or `elements::` prefix. Ditto for `BlockHash` though it's less of a problem since the two kinds of blockhashes are basically the same thing.

Reply:
Ok, this is a convention I didn't know about. Good to know and I will make sure to follow it from now.

Reply:
It's mostly just a convention for Elements software, which always needs to deal with both kinds of objects.